lib/ssh_scan/os/ubuntu.rb

```diff
@@ -1,12 +1,187 @@
 module SSHScan
   module OS
     class Ubuntu
+      @@fingerprints = {
+       "warty"=> ["3.8.1p1-11ubuntu3.3", "3.8.1p1-11ubuntu3.2", "3.8.1p1-11ubuntu3"],
```

Comment:
I think it would be more practical if the key here was an Ubuntu numbered version.